lib/pages/tab.dart

```diff
@@ -116,6 +116,7 @@ class _TabPageState extends State<TabPage> {
         hasInk: true,
         inkColor: Color.fromRGBO(0, 0, 0, .15),
         onTap: (index) => setState(() => _currentIndex = index),
+        borderRadius: BorderRadius.vertical(top: Radius.circular(12)),
```

Comment:
A borderRadius is unnecessary when the BottomNavigation widget doesn't have a different background color or shadow. In this case, it only results in an off looking shadow when using light theme on the trading post tab.

Reply:
This was a change I was playing with that I missed reverting.